clang_delta/TransformationManager.h

```diff
@@ -11,6 +11,8 @@
 #ifndef TRANSFORMATION_MANAGER_H
 #define TRANSFORMATION_MANAGER_H
 
+#define CREDUCE_LIBCLC_INCLUDE_PATH "CREDUCE_LIBCLC_INCLUDE_PATH"
```

Comment:
Seems the macro is only used in the TransformationManager.cpp. Can we define it locally rather than exposing it to other files. Or, we could just use getenv("CREDUCE_LIBCLC_INCLUDE_PATH"). Looks like the macro definition is verbose here?